index.d.ts

```diff
@@ -714,3 +714,12 @@ interface DurableObjectNamespace {
 
   get: (id: DurableObjectId) => DurableObjectStub;
 }
+
+interface WebSocket {
+  accept(): void;
+}
+
+declare class WebSocketPair {
```

Comment:
To be able to deconstruct the websocket pair via:
```js
let [client, server] = new WebSocketPair();
```
Wouldn't there need to be an extend to `Array`? Or implementing a symbol iterator.
```suggestion
declare class WebSocketPair extends Array {
```

Reply:
`WebSocketPair` isn't iterable... I tried :(

Reply:
The docs mention it is "common" to destructure `Object.values(pair)` but it's shorter to simply destructure the object `const { 0: client, 1: server } = pair`

Reply:
Ah.. my bad 😅 Thanks for the clarification!

Reply:
No problem! I had the exact same assumption because the rest of the DX around cf workers and durable objects is lovely -- this was an exception!